### PR TITLE
Doc 8594b

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/cli/pages/mctimings.adoc [hunk 1]
```diff
@@ -93,501 +73,331 @@ If using a secure connection (the `-s` flag) then this should be 11211.
...
-[ 640 -  649]us ( 99.57%)     1 |
-[ 750 -  759]us ( 99.61%)     1 |
-[ 800 -  809]us ( 99.68%)     2 |
-[ 860 -  869]us ( 99.71%)     1 |
-[ 870 -  879]us ( 99.75%)     1 |
-[   1 -    1]ms ( 99.93%)     5 |
-[  32 -   32]ms ( 99.96%)     1 |
-[  39 -   39]ms (100.00%)     1 |
-Total: 2805 operations
+[  0.00 -   6.00]us (0.0000%)	   2|
```
Comment (rdemellow) [inline modules/cli/pages/mctimings.adoc:266]: We would now print the legend too before the first histogram is rendered. As the -v flag is set.

File: modules/cli/pages/mctimings.adoc [hunk 2]
```diff
@@ -93,501 +73,331 @@ If using a secure connection (the `-s` flag) then this should be 11211.
...
 | `-v`
-| Show operation timings as histograms.
-If this flag is used, one or more operation names or associated hexadecimals can be specified; so as to limit output to that which corresponds to those operations.
-Hexadecimals must be specified within _back-ticks_.
-If this flag is not used, then only the total number of operations are displayed.
+| Specifies that operation timings are displayed as histograms.
+
+Whenever `-v` is used, one or more operation names can also be specified; so as to restrict output to that corresponding to the specified operations.
+
+If `-v` is _not_ used, the displayed output consists only of a _total_ figure for each of all operations performed.
```
Comment (rdemellow) [inline modules/cli/pages/mctimings.adoc]: This flag is also overloaded for when the --json argument is used. -v will pretty print the json output.

**Comment by tonyjhillman** [comment]
I've shown the legend, and have also reworked things to make the -j option more prominent.

**Comment by tonyjhillman** [comment]
Closing this PR and substituting https://github.com/couchbase/docs-server/pull/2001 instead. The new PR will contain all the changes made here, and more.

**Pull request closed without merging**